src/Component/AboutPlace/index.js

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import { styled } from '@mui/system';
+import IconsDetailPage from '../IconsDetailPage';
+import StartBooking from '../StartBooking';
+export default function AboutPlace() {
+    return (
+        <Box sx={{ position: 'relative', marginTop: 5 }}>
```

Comment:
I think there is no need for `position: relative`, please remove it if there is no effect for it